12 MolecularGlassOrientationMapping.ipynb

```diff
@@ -0,0 +1,1038 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.            data_smooth = gaussian_filter1d(data,sig)
</pre><p>Think sig should be sigma here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyxem/pyxem-demos/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>